.circleci/config.yml

```diff
@@ -194,12 +191,22 @@ jobs:
 
   lint:
     executor: node
-    parallelism: 2
     steps:
-      - <<: *attach_to_bootstrap
+      - checkout
+      - run:
+          name: "remove workspaces from package.json"
```

Comment:
Why?

Reply:
Our workspace install is super slow, for eslint/prettier we don't need all packages, just the root ones. This allows us to start linting when we're bootstrapping, this will give us feedback faster for linting. This way we can also shave a minute from persisting the workspace twice in bootstrap.